launch/client.py

```diff
@@ -486,6 +496,86 @@ def create_model_endpoint(
                     "Endpoint should be one of the types 'sync' or 'async'"
                 )
 
+    def deploy_service(
```

Comment:
This seems at first glance to be a "higher abstraction level" than `create_model_endpoint`, do we want to put this in its own file and have `client.py` handle all the low-level calls? (Purely a "code organization" question)